test/contrib/test_ssltransport.py

```diff
@@ -0,0 +1,402 @@
+from dummyserver.testcase import SocketDummyServerTestCase, consume_socket
+from dummyserver.server import (
+    DEFAULT_CERTS,
+    DEFAULT_CA,
+)
+
+from urllib3.contrib.ssl import SSLTransport
+
+import select
+import pytest
+import socket
+import ssl
+import sys
+
+
+def server_client_ssl_contexts():
+    if hasattr(ssl, "PROTOCOL_TLS_SERVER"):
+        server_context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
+    else:
+        server_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
```

Comment:
Can we mention that this is only for Python 2.7 and 3.5? We'll remove them in a few months, and even if coverage will show the missing lines, it will help with the grep work.

Reply:
good idea.